drivers/sensor/icm42670S/Kconfig

```diff
@@ -33,6 +33,26 @@ config ICM42670S_APEX_WOM
 	bool "Wake on Motion (WoM): Detects motion when accelerometer samples exceed a programmable threshold"
 endchoice
 
+config ICM42670S_AML
+	bool "ICM42670S Air Motion Library (AML)"
+	default n
+
+choice AML_FEATURES
```

Comment:
2 comments:
- Might be useful to explain that this controls what AML output is printed
- Could be interesting to allow multiple output at the same time

Reply:
I think it's a wonderful idea, please proceed as you suggested.